### ISSUE TITLE
Documentation keeps referring to "the new/old API"

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#11402]
|[<img alt="jyknight's avatar" src="https://avatars.githubusercontent.com/u/873903?s=50" width="50" height="50">](https://github.com/jyknight)| @jyknight reported|
|-|-|
|Trac ID|trac#219|
|Type|defect|
|Created|2003-08-22 01:20:41Z|

<details><summary>Searchable metadata</summary>

```
trac-id__219 219
type__defect defect
reporter__jknight jknight
priority__high high
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__ 
keywords__documentation documentation
time__1061515241000000 1061515241000000
changetime__1468028949268917 1468028949268917
version__ 
owner__ 
cc__spiv cc__jknight
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="jyknight's avatar" src="https://avatars.githubusercontent.com/u/873903?s=50" width="50" height="50">](https://github.com/jyknight)|@jyknight commented|
|-|-|

```
#!html
<pre>
I assume that this is referring to pre 1.0.0 twisted? If so, references to it 
should be completely excised from the main documentation (and 
relegated to a "API Changes" section, if anything), because that is a 
looong time ago. If I'm wrong and it's more recent, old/new should be 
given a version number.

e.g. """With the new API, Protocols no longer connect directly using 
reactor.client*""", section 3.4.3. There were a few other examples earlier 
too.

</pre>
```

**Comment by twisted-trac** [comment]
|[<img alt="moshez's avatar" src="https://avatars.githubusercontent.com/u/422703?s=50" width="50" height="50">](https://github.com/moshez)|@moshez commented|
|-|-|

```
#!html
<pre>
Checking in clients.html;
/cvs/Twisted/doc/howto/clients.html,v  &lt;--  clients.html
new revision: 1.10; previous revision: 1.9
done

Fixed.
[There doesn't seem to be 'lots' of examples, just this one. If you find more,
please open new bugs for them. I tried to grep for old/new and manually go
through the hits, and didn't find anything. Thanks.]
</pre>
```

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_4"></a>|GitHub commented|
|-|-|

In changeset 8fe97b3f224ebb5038f9f8873d48dd38b422c64e
```
#!CommitTicketReference repository="" revision="8fe97b3f224ebb5038f9f8873d48dd38b422c64e"
Merge pull request #219 from twisted/8453-rodrigc-types

Author: rodrigc
Reviewer: adiroiban
Fixes: #8453

Replace use of types constants with primitive types in twisted/mail
```
